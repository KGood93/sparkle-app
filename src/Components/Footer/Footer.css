.footer {
    left: 0;
    bottom: 0;
    width: 100%;
    height: 40px;
    background-color: rgb(241,221,210);
    border-top: solid 1px rgb(6,106,117);
    text-align: center;
}

.footerText {
    font-family: 'Raleway', sans-serif;
    font-size: 10px;
    margin-top: 0px;
    margin-bottom: 0px;
    color: rgb(6,106,117);
}

.footerLink:link {
    color: rgb(6,106,117);
}
  
.footerLink:visited {
    color: rgb(6,106,117);
}
  
.footerLink:hover {
    color: rgb(251,159,139);
}
  
.footerLink:active {
    color: rgb(251,159,139);
}