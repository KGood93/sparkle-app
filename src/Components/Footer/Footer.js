import React, { Component } from 'react'
import './Footer.css'

class Footer extends Component {
    render() {
        return (
            <footer className="footer">
                <p className="footerText">Kathryn A. Goodreau</p>
                <p className="footerText">E-mail: <a href="mailto:goodreau.kath@gmail.com?Subject=Hello" className="footerLink">goodreau.kath@gmail.com</a></p>
                <p className="footerText">Links of Interest: <a href="https://github.com/KGood93" className="footerLink" target="_blank" rel="noopener noreferrer">Github</a> and <a href="https://www.linkedin.com/in/kathrynagoodreau/" className="footerLink" target="_blank" rel="noopener noreferrer">LinkedIn</a></p>
            </footer>
        )
    }
}

export default Footer