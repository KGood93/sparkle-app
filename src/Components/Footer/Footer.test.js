import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe('Footer Component', () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<Footer />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});