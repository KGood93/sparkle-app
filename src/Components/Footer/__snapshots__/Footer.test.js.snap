// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders without crashing 1`] = `
<footer
  className="footer"
>
  <p
    className="footerText"
  >
    Kathryn A. Goodreau
  </p>
  <p
    className="footerText"
  >
    E-mail: 
    <a
      className="footerLink"
      href="mailto:goodreau.kath@gmail.com?Subject=Hello"
    >
      goodreau.kath@gmail.com
    </a>
  </p>
  <p
    className="footerText"
  >
    Links of Interest: 
    <a
      className="footerLink"
      href="https://github.com/KGood93"
      rel="noopener noreferrer"
      target="_blank"
    >
      Github
    </a>
     and 
    <a
      className="footerLink"
      href="https://www.linkedin.com/in/kathrynagoodreau/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
    </a>
  </p>
</footer>
`;
